Gruntfile.js

```diff
@@ -28,9 +30,10 @@ module.exports = function(grunt) {
 
         sass: {
             options: {
+                implementation: sass,
                 sourceMap: true
             },
-            interactive: {
+            dist: {
```

Comment:
What's the difference here?

Reply:
Sure. The problem was that an old version of grunt sass was being used. So when I updated grunt, and then grunt-sass I got some errors so looking at the latest version of grunt-sass - I saw some of the syntax had changed. So pasted that in and it worked 🙏 
https://github.com/sindresorhus/grunt-sass#usage